src/main/java/it/unibo/alchemist/boundary/gui/layouts/AFlowLayout.java

```diff
@@ -1,221 +1,222 @@
-        if (isOrdered()) {
-            getComponentsList().add(comp);
-        }
-    }
-
-    /**
-     * 
-     * @return
-     */
-    protected int getAlignment() {
-        return alignment;
-    }
-
-    /**
-     * 
-     * @return
-     */
-    protected int getAnchor() {
-        return anchor;
-    }
-
-    /**
-     * 
-     * @param c
-     * @return
-     */
-    public int getComponentOrder(final Component c) {
-        if (isOrdered()) {
-            return compOrder.indexOf(c);
-        } else {
-            final int n = c.getParent().getComponentCount();
-            final Component[] components = c.getParent().getComponents();
-            for (int i = 0; i < n; i++) {
-                if (components[i].equals(c)) {
-                    return i;
-                }
-            }
-            return -1;
-        }
-    }
-
-    /**
-     * 
-     * @return
-     */
-    public List<Component> getComponentsList() {
-        return compOrder;
-    }
-
-    /**
-     * 
-     * @return
-     */
-    protected int getGap() {
-        return gap;
-    }
-
-    /**
-     * 
-     * @return
-     */
-    public boolean isOrdered() {
-        return compOrder != null;
-    }
-
-    @Override
-    public abstract void layoutContainer(final Container parent);
-
-    /**
-     * 
-     * @param parent
-     * @param minimum
-     * @return
-     */
-    protected abstract Dimension layoutSize(final Container parent, final boolean minimum);
-
-    @Override
-    public Dimension minimumLayoutSize(final Container parent) {
-        return layoutSize(parent, false);
-    }
-
-    @Override
-    public Dimension preferredLayoutSize(final Container parent) {
-        return layoutSize(parent, false);
-    }
-
-    @Override
-    public void removeLayoutComponent(final Component comp) {
-        if (isOrdered()) {
-            getComponentsList().remove(comp);
-        }
-    }
-
-    /**
-     * 
-     * @param alignment
-     */
-    protected void setAlignment(final int alignment) {
-        this.alignment = alignment;
-    }
-
-    /**
-     * 
-     * @param anchor
-     */
-    protected void setAnchor(final int anchor) {
-        this.anchor = anchor;
-    }
-
-    /**
-     * 
-     * @param c
-     * @param order
-     * @throws IllegalStateException
-     */
-    public void setComponentOrder(final Component c, final int order) throws IllegalStateException {
-        if (!isOrdered()) {
-            throw new IllegalStateException();
-        }
-        final Component temp = compOrder.get(order);
-        final int old = compOrder.indexOf(c);
-        compOrder.set(order, c);
-        compOrder.set(old, temp);
-    }
-
-    /**
-     * 
-     * @param gap
-     */
-    protected void setGap(final int gap) {
-        this.gap = gap;
-    }
-
-    @Override
-    public String toString() {
-        return getClass().getName() + "[gap=" + gap + " align=" + alignment + " anchor=" + anchor + "]";
-    }
-}
+/*

```

Comment:
There is something I am not understanding, it looks like being the same file. Please take a look on what's happened here, I fear it's the \n thing again